index.html

```diff
@@ -1328,6 +1321,118 @@
+          </h3>
+          <pre class="idl">
+            [Constructor(DOMString type, optional PresentationSessionConnectEventInit eventInitDict)]
+            interface PresentationSessionConnectEvent : Event {
+              [SameObject] readonly attribute PresentationSession session;
+            };
+
+            dictionary PresentationSessionConnectEventInit : EventInit {
+              PresentationSession session;
+            };
+
+</pre>
+          <p>
+            An event named <code>sessionconnect</code> is fired on a
+            <a>PresentationRequest</a> when a session associated with the
+            object is initiated. It is fired at the <a>PresentationRequest</a>
+            instance, using the <a>PresentationSessionConnectEvent</a>
+            interface, with the <a for=
+            "PresentationSessionConnectEvent">session</a> attribute set to the
+            <a><code>PresentationSession</code></a> object that was initiated.
+            The event is fired regardless of how the session was initiated. For
+            example, it can come from a call to <code>start()</code>,
+            <code>join()</code> or even from a UA initiated session on <a for=
+            "Presentation"><code>defaultRequest</code></a>.
+          </p>
+          <p>
+            The UA MUST fire the event as soon as it can create the
+            <a><code>PresantiotSession</code></a> associated to the event.
+          </p>
+        </section>
+      </section>
+      <section>
+        <h3>
+          Interface <a><code>Presentation</code></a>
+        </h3>
+        <pre class="idl">
+          partial interface Navigator {
+            [SameObject] readonly attribute Presentation presentation;
+          };
+
+</pre>
+        <p>
+          The <dfn for="Navigator"><code>presentation</code></dfn> attribute is
+          used to retrieve an instance of the <a>Presentation</a> interface.
+        </p>
+        <pre class="idl">
+          interface Presentation : EventTarget {
+            // This API used by controlling browsing context.
+            attribute PresentationRequest? defaultRequest;
+
+            // This API used by presenting browsing context.
+            Promise&lt;PresentationSession&gt; getSession();
+            Promise&lt;sequence&lt;PresentationSession&gt;&gt; getSessions();
+            attribute EventHandler onsessionavailable;
+          };
+
+</pre>
+        <p>
+          The <dfn for="Presentation"><code>defaultRequest</code></dfn> MUST
+          return the <a>default presentation request</a> if any, null
+          otherwise.
+        </p>
+        <section>
+          <h4>
+            Default presentation request
+          </h4>
+          <p>
+            If set, the <a for="Presentation">defaultRequest</a> SHOULD be used
+            by the UA as the <dfn>default presentation request</dfn>. When the
+            UA wants to start or join a session from its UI, it MUST use the
```

Comment:
I would prefer this as SHOULD for two reasons:
  - There are use cases for the user or UA overriding this behavior.
  - On some UAs with limitations on the user interface, it may not be possible to initiate a presentation from the UA at all.